ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -39,6 +39,7 @@ component:
   dimmer: "A device which can reduce lighting output of a lamp (thus 'dimming' the light)."
   dishwasher: "Device that washes dishes."
   dryer: "A component used for drying clothes."
+  elevator: "Machine to vertically transport people or items between floors in the building."
```

Comment:
There is no need to define `elevator` as a subfield since the idea of an "elevator" is represented by the entity type `ELV`. For what you are trying to do I would use `run_status`.